### DISCUSSION TITLE
为什么用macos协议登录上了，还是登录失败

### DISCUSSION THREAD

**Discussion by mumangguo** [simple-robot/Discussions#117] [打扰了]
### 涉及的编程语言

Java

### 疑问描述

[![CDD08C53C76F430922483C4FA51006B1](https://cdn.jsdelivr.net/gh/mumangguo/images@master/20230216/CDD08C53C76F430922483C4FA51006B1.g200bmo09c0.jpg)](https://raw.githubusercontent.com/mumangguo/images/master/20230216/CDD08C53C76F430922483C4FA51006B1.g200bmo09c0.jpg)
![4715D6854D4E73C7AF54BF7EAF3E37CA](https://cdn.jsdelivr.net/gh/mumangguo/images@master/20230216/4715D6854D4E73C7AF54BF7EAF3E37CA.78vbdz13b0g0.jpg)
![3](https://cdn.jsdelivr.net/gh/mumangguo/images@master/20230216/3.6to1x7ioga40.png)
![4](https://cdn.jsdelivr.net/gh/mumangguo/images@master/20230216/4.1sjkivbi6eww.png)
想问一下要不要改这个配置文件啊，还是怎么搞

**Comment by ForteScarlet** [comment]
配置默认的协议为安卓协议，如果要修改参考[文档](https://component-mirai.simbot.forte.love/docs/bot-config)中 `config.protocol` 说明。
如果仍有异常或预期外的结果，请反馈到 [issues](https://github.com/simple-robot/simpler-robot/issues/new?assignees=ForteScarlet&labels=bug%3F&template=bug-report.yml) 。

**Reply by mumangguo** [reply]
![2](https://cdn.jsdelivr.net/gh/mumangguo/images@master/20230216/2.6ss1w64q3yo0.png)
![1](https://cdn.jsdelivr.net/gh/mumangguo/images@master/20230216/1.2z32uovvkbc0.png)
不知道是不是这么配

**Reply by ForteScarlet** [reply]
- https://simbot.forte.love/docs/quick-start/spring-boot-starter#%E9%85%8D%E7%BD%AE
- https://simbot.forte.love/docs/basic/bot-config
- https://component-mirai.simbot.forte.love/docs/bot-config/

**Comment by mumangguo** [comment]
![1](https://cdn.jsdelivr.net/gh/mumangguo/images@master/20230216/1.30mejydd9xa.png)
![2](https://cdn.jsdelivr.net/gh/mumangguo/images@master/20230216/2.1jr7e138if0g.png)
还是一样的报错。。。

**Reply by ForteScarlet** [reply]
重复截取10行不到的半截日志得不到任何有效信息。
任何与预期不符、疑似bug或有异常的错误请反馈到 [issues](https://github.com/simple-robot/simpler-robot/issues/new?assignees=ForteScarlet&labels=bug%3F&template=bug-report.yml)

**Reply by mumangguo** [reply]
你看看配置对了吗，对了的话，我自己再看看

**Reply by ForteScarlet** [reply]
登录协议的配置内容应该没有什么问题。

- 更多无法登录的问题汇总与解决可参考 https://github.com/mamoe/mirai/issues/2570
